content/docs/dev/concepts/explore-tikv-features/br/backup-restore.md

```diff
@@ -75,15 +79,15 @@ Explanations for some options in the above command are as follows:
 - `128`: The value of `ratelimit`, unit is MiB/s.
 - `--pd`: Service address of `PD`.
 - `"${PDIP}:2379"`:  Parameter of `--pd`.
-- `--dst-api-version`: The `api-version`, please see [API V2](../api-v2).
+- `--dst-api-version`: The `api-version`, please see [API V2].
 - `v2`: Parameter of `--dst-api-version`, the optionals are `v1`, `v1ttl`, `v2`(Case insensitive). If no `dst-api-version` is specified, the `api-version` is the same with TiKV cluster of `--pd`.
-- `gcttl`: The pause duration of GC. This can be used to make sure that the incremental data from the beginning of backup to TiKV-CDC [create changefeed](../cdc) will NOT be deleted by GC. 5 minutes by default.
+- `gcttl`: The pause duration of GC. This can be used to make sure that the incremental data from the beginning of backup to [Create a replication task] will NOT be deleted by GC. 5 minutes by default.
```

Comment:
```suggestion
- `gcttl`: The pause duration of GC. This can be used to make sure that the incremental data from the beginning of backup to [Creating a replication task] will NOT be deleted by GC. 5 minutes by default.
```